### PR TITLE
docs(html): rewrite readme

### PR DESCRIPTION
**Pull request by HonkingGoose**
@coderabbitai: ignore

## Changes

- Be specific about what things the `html` manager can update
- Link to the cdnjs CDN homepage
- Explain key differences between `html` and `cdnurl` manager

## Context

Drive-by PR, not closing any issue with this.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/manager/html/readme.md [hunk 1]
```diff
@@ -1 +1,7 @@
-The `html` manager extracts scripts and CSS includes pointing to cdnjs. It differs from the `cdnjs` manager in that it detects if an integrity hash is already present and updates it too if found.
+The `html` manager updates `<script>` tags and _CSS includes_ that point to the [cdnjs content delivery network](https://cdnjs.com/).
```
Comment (HonkingGoose) [inline lib/modules/manager/html/readme.md:1]: Just saying _CSS includes_ is too vague. What's the correct CSS syntax?

File: lib/modules/manager/html/readme.md [hunk 2]
```diff
@@ -1 +1,7 @@
...
+It also updates Subresource Integrity (SRI) hashes in `integrity` attributes.
+
+Key differences between the `cdnurl` manager and the `html` manager:
```
Comment (HonkingGoose) [inline lib/modules/manager/html/readme.md:4]: I think the old `cdnjs` manager was renamed/migrated to the [`cdnurl` manager](https://docs.renovatebot.com/modules/manager/cdnurl/)?

We do still have a `cdnjs` _datasource_. Is that what we meant?

File: lib/modules/manager/html/readme.md [hunk 3]
```diff
@@ -1 +1,7 @@
-The `html` manager extracts scripts and CSS includes pointing to cdnjs. It differs from the `cdnjs` manager in that it detects if an integrity hash is already present and updates it too if found.
+The `html` manager updates `<script>` tags and _CSS includes_ that point to the [cdnjs content delivery network](https://cdnjs.com/).
+It also updates Subresource Integrity (SRI) hashes in `integrity` attributes.
+
```
Comment (HonkingGoose) [inline lib/modules/manager/html/readme.md:3]: Can one of the maintainers read the `html` code for me, and check if the docs match the code? I'm bad at reading the code. 🙈

**Review by HonkingGoose — COMMENTED** [review]
Questions for the maintainers.

File: lib/modules/manager/html/readme.md (hunk 1 above)
Comment (rarkins) [reply lib/modules/manager/html/readme.md:1]: We don't really care about the syntax - we just look for cdnjs URLs

Comment (HonkingGoose) [reply lib/modules/manager/html/readme.md:1]: If we don't care about the syntax, then we don't need to mention it. 😄

**Review by viceice — APPROVED** [review]

**Pull request merged**